public/src/js/models/article/editor.js

```diff
@@ -56,16 +56,24 @@ export default class Editor extends BaseClass {
             return opts.maxLength && this.value().length > opts.maxLength;
         });
         this.displayEditor = ko.pureComputed(this.isVisible, this);
-        this.updateText = ko.pureComputed({
-            read: this.value,
-            write: this.writeText,
-            owner: this
-        });
+        if (type === 'text') {
```

Comment:
can you group all the `if (type ===` together after line 49, all the type dependent logic should be in the same place